packages/victory-core/src/victory-util/add-events.tsx

```diff
@@ -68,9 +68,9 @@ export interface EventsMixinClass<TProps> {
     defaultAnimationWhitelist: string[],
   ): React.ReactElement;
   getComponentProps(
-    component: React.ReactElement,
+    component: React.ReactNode,
```

Comment:
Used a more inclusive type for the component, as it can also be a `string` or `string[]`